### ISSUE TITLE
New praatio version 5.0 will break things

### ISSUE THREAD

**Issue by timmahrt** [dlce-eva/pyvoices#4]
Hi, I'm the maintainer of praatio, a dependency used in your code.  It's not released yet, but Praatio 5.0 will break some things.  The changes to praatio aren't so heavy, but if you want to keep things as they are, you can change
```
install_requires=["praatio"],
```
to
```
install_requires=["praatio ~= 4.3"],
```
in your setup.py file.

Sorry for the inconvenience!

**Comment by xrotwang** [comment]
@timmahrt thanks for the heads-up! We'll try to add enough tests to figure out whether we can work with Praatio 5, rather than pinning an older version.

**Comment by Bibiko** [comment]
resolved with 9b58762b99c67f1e46b8975335e7f71dea6fa159
